src/Mobs/Monster.cpp

```diff
@@ -1129,7 +1135,6 @@ bool cMonster::WouldBurnAt(Vector3d a_Location, cChunk & a_Chunk)
 		(a_Chunk.GetSkyLight(RelX, RelY, RelZ) == 15) &&             // In the daylight
```

Comment:
We could prevent this with a macro, instead of calling super::tick, make two macros such as:
`SUPER_TICK_AT_START();`
and 
`SUPER_TICK_END()`
The former first does the chunk update thing, and only then calls super, while the latter just calls super.